lib/ipsecbase.pm

```diff
@@ -112,10 +112,11 @@ sub pre_run_hook {
     $self->SUPER::pre_run_hook;
 }
 
-sub ensure_apparmor_disabled () {
-    unless (systemctl "is-active apparmor", ignore_failure => 1) {    # 0 if active, unless to revert
-        systemctl "disable --now apparmor";
-        record_info "apparmor", "disabled";
+sub ensure_service_disabled {
```

Comment:
Looks like generic function. It would make sense to put it in some utils. @dzedro could you please recommend some place?